ci/util/fbuild_compiledb.py

```
@@ -0,0 +1,84 @@
+            ``esp32c2``, matching the ``-e`` flag on fbuild).
+
+    Returns:
+        Path to ``compile_commands.json`` on success, ``None`` otherwise.
+    """
+    release = fbuild_release_dir(build_root, board_name)
+    cdb = release / "compile_commands.json"
+    if cdb.exists():
+        return cdb
+
+    try:
+        subprocess.run(
```

Ruff (0.15.10)

[error] `subprocess` call: check for execution of untrusted input

(S603)